desk/app/bait.hoon

```diff
@@ -72,10 +77,22 @@
   ^-  (quip card _this)
   =/  old  !<(versioned-state old-state)
   ?-  -.old
-      %1
+      %2
     `this(state old)
+  ::
+      %1
+    =/  new-metadata
+      %-  ~(gas by *(map cord metadata:reel))
+      %+  turn
+        ~(tap by token-metadata.old)
+      |=  [[inviter=ship token=cord] meta=metadata:reel]
```

Comment:
```suggestion
    =/  new-metadata
      %-  ~(gas by *(map token:reel metadata:reel))
      %+  turn
        ~(tap by token-metadata.old)
      |=  [[inviter=ship =token:reel] meta=metadata:reel]
```


(Although I think here you may _technically_ be able to get away with `~(gas by token-metadata.state)`...)